.github/workflows/tests.yml

```diff
@@ -21,15 +21,10 @@ jobs: # Docs: <https://git.io/JvxXE>
         setup: [basic, lowest]
         laravel: [default]
         coverage: [yes]
-        php: ['7.3', '7.4', '8.0']
-        include:
-          - php: '7.3'
-            setup: basic
-            coverage: no
-            laravel: '^7.0'
+        php: ['8.1','8.2','8.3']
```

Comment:
@eldario 

>This scheduled workflow is disabled because there hasn't been activity in this repository for at least 60 days. 

Please, enable the workflow https://github.com/avtocod/b2b-api-php-laravel/actions/workflows/tests.yml